e3db/config.py

```diff
@@ -12,7 +12,7 @@ class Config:
 
     DEFAULT_API_URL = "https://api.e3db.com"
 
-    def __init__(self, client_id, api_key_id, api_secret, public_key, private_key, client_email="", version="1", api_url=DEFAULT_API_URL):
+    def __init__(self, client_id, api_key_id, api_secret, public_key, private_key, public_signing_key, private_signing_key, client_email="", version="1", api_url=DEFAULT_API_URL):
```

Comment:
if signing keys are passed in we should set the version to 2, and the default version should be 2 now (v1 clients didn't have signing keys, v2 clients do, thats the only difference)

Reply:
Defaults to 2 now in config init, and updated client backup method to default to 2